docs/kqueen.rst

```diff
@@ -364,43 +364,46 @@ You can provision a Kubernetes cluster using various community of engines, such
+#. Click ``Submit``.
+#. To track the cluster status, navigate to the KQueen main dashboard.
 
 To provision a Kubernetes cluster using the Azure Kubernetes Service:
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-1. Log in to https://portal.azure.com.
-2. Create an Azure Active Directory Application as described in the official Microsoft `Documentation <https://docs.microsoft.com/en-us/azure/azure-resource-manager/resource-group-create-service-principal-portal#create-an-azure-active-directory-application>`_.
-3. Copy the Application ID, Application Secret, Tenant ID (Directory ID), and Subscription ID to use in step 8.
-4. Set the ``Owner`` role to your Application in the Subscription settings to enable the creation of Kubernetes clusters.
-5. Navigate to the ``Resource groups`` tab and create a resource group. Copy the  ‘Resource group name’ to use in step 8.
-6. From the to ``Resource groups`` -> your_group -> Access Control (IAM) tab, verify that the Application has the ``Owner`` role in Resource group.
-7. Log in to the KQueen web UI.
-8. From the ``Create provisioner`` tab, select the AKS engine and set the following:
-	1. Set the ``Client ID`` as Application ID from step 3.
-	2. Set the ``Resource group name`` as Resource group name from step 4.
-	3. Set the ``Secret`` as Application Secret from step 3.
-	4. Set the ``Subscription ID`` as  Subscription ID from step 3.
-	5. Set the ``Tenant ID`` as Tenant(Directory) ID from step 3.
-9. In the KQueen web UI, click ``Deploy Cluster``.
-10. Select the AKS provisioner.
-11. Specify the cluster requirements.
-12. Specify the public SSH key to connect to AKS VM’s. For ssh access into created VM’s, assign the public IP address to the VM as described in `guide <https://gist.github.com/naumvd95/576d6e48200597ca89b26de15e8d3675>`_). Once done, use foollowing command: ``ssh azureuser@<<public_ip>> -i .ssh/your_defined_id_rsa``.
-13. Click ``Submit``.
-14. To track the cluster status, navigate to the KQueen main dashboard.
+#. Log in to `Azure Kubernetes Service <https://portal.azure.com>`_.
+#. Create an Azure Active Directory Application as described in the official Microsoft `Documentation <https://docs.microsoft.com/en-us/azure/azure-resource-manager/resource-group-create-service-principal-portal#create-an-azure-active-directory-application>`_.
+#. Copy the Application ID, Application Secret, Tenant ID (Directory ID), and Subscription ID to use in step 8.
+#. Set the ``Owner`` role to your Application in the Subscription settings to enable the creation of Kubernetes clusters.
+#. Navigate to the ``Resource groups`` tab and create a resource group. Copy the  ‘Resource group name’ to use in step 8.
+#. From the to ``Resource groups`` -> your_group -> Access Control (IAM) tab, verify that the Application has the ``Owner`` role in Resource group.
+#. Log in to the KQueen web UI.
+#. From the ``Create provisioner`` tab, select the AKS engine and set the following:
```

Comment:
... the AKS engine.
You already use set in the next steps, so it's not needed here

Reply:
... the ``Azure Managed Kubernetes Engine``.

Reply:
done